include/image-commands.mk

```diff
@@ -229,6 +229,19 @@ define Build/combined-image
 	@mv $@.new $@
 endef
 
+define Build/combined-ext-image
```

Comment:
It is openmesh specific, isn't it? Please use a name which indicates it.

Reply:
It was never planned to be OpenMesh specific but it looks like it ended up that way. What about openmesh-ce as name?

Reply:
Or just openmesh-image, following the naming of already existing recipes in the file.